android-core/src/main/java/com/mparticle/networking/MParticleBaseClientImpl.java

```diff
@@ -110,6 +111,8 @@ protected MPUrl getUrl(Endpoint endpoint, @Nullable String identityPath, boolean
             url = domainMappingUrl;
         }
         Uri uri;
+        if(endpoint!=Endpoint.CONFIG){
```

Comment:
```suggestion
        if (endpoint != Endpoint.CONFIG ) {
```
Nitpick, we should fix the spacing here.